### PR TITLE
tune prose in text that references [HTTP] for status code information (#915)

### PR DESCRIPTION
**Pull request by reschke**

### CODE REVIEW

**Review by royfielding — CHANGES_REQUESTED** [review]
I don't think this is correct. I can try rewriting the whole sentence, if necessary.

**Comment by reschke** [comment]
Please do :-)

**Review by royfielding — APPROVED** [review]

File: draft-ietf-httpbis-messaging-latest.xml [hunk 1]
```diff
@@ -821,16 +821,20 @@ http://www.example.org:8080
...
-   corresponding request. The rest of the response message is to be
-   interpreted in light of the semantics defined for that status code.
-   See <xref target="status.codes"/> for information about the semantics of status codes,
-   including the classes of status code (indicated by the first digit),
-   the status codes defined by this specification, considerations for the
-   definition of new status codes, and the IANA registry.
+   corresponding request. A recipient parses and interprets the remainder
+   of the response message in light of the semantics defined for that
+   status code, if it is understood, or in accordance with the class of
+   that status code indicated by its first digit.
```
Comment (mnot) [inline draft-ietf-httpbis-messaging-latest.xml:827]:
```suggestion
   that status code indicated by its first digit if it is not.
```

**Review by mnot — APPROVED** [review]
suggestion

**Pull request merged**